.NET/Microsoft.Recognizers.Text.DateTime/Models/DateTimeModel.cs

```diff
@@ -32,33 +32,24 @@ public List<ModelResult> Parse(string query, DateObject refTime)
             // Preprocess the query
             query = QueryProcessor.Preprocess(query);
 
-            try
+            var extractResults = Extractor.Extract(query, refTime);
+
+            foreach (var result in extractResults)
             {
-                var extractResults = Extractor.Extract(query, refTime);
+                var parseResults = Parser.Parse(result, refTime);
 
-                foreach (var result in extractResults)
+                if (parseResults.Value is List<DateTimeParseResult> list)
                 {
-                    var parseResults = Parser.Parse(result, refTime);
-
-                    if (parseResults.Value is List<DateTimeParseResult> list)
-                    {
-                        parsedDateTimes.AddRange(list);
-                    }
-                    else
-                    {
-                        parsedDateTimes.Add(parseResults);
-                    }
+                    parsedDateTimes.AddRange(list);
+                }
+                else
+                {
+                    parsedDateTimes.Add(parseResults);
                 }
-
-                // Filter out ambiguous cases. Naïve approach.
-                parsedDateTimes = Parser.FilterResults(query, parsedDateTimes);
-            }
-            catch (Exception)
-            {
-                // Nothing to do. Exceptions in parse should not break users of recognizers.

```

Comment:
I'm a bit concerned about removing a try catch IF the Parser can throw an exception.

Reply:
I think, the PR will be approved maintaining the try/catch structure, removing the unused code if any.
Add a new task to review this specific smelling code: _Try/Catch in code without any explicit action_
Also, add a new issue in our repo to review. And then create the issue in MS repo to socialize and receive feedback and or approval to tackle it.